docs/download.md

```diff
@@ -18,6 +18,72 @@ cuDF jar, that is either preinstalled in the Spark classpath on all nodes or sub
 that uses the RAPIDS Accelerator For Apache Spark. See the [getting-started
 guide](https://nvidia.github.io/spark-rapids/Getting-Started/) for more details.
 
+## Release v21.12.0
+Hardware Requirements:
+
+The plugin is tested on the following architectures:
```

Comment:
maybe a nit and again fixed for later, but aren't the "architectures" really Ampere, Volta, Turing vs GPU models?